### PR TITLE
[JAV-364]add more details message when laodbalance not configured in …

### PR DESCRIPTION
**Pull request by liubao68**
…consumer chain. add thread id in console log.

### CODE REVIEW

**Comment by eric-lee-ltk** [comment]
Can this warning show up when the application starts instead of runtime?

File: core/src/main/java/io/servicecomb/core/Invocation.java [hunk 1]
```diff
@@ -74,6 +74,9 @@ private void init(OperationMeta operationMeta, Object[] swaggerArguments) {
   }
 
   public Transport getTransport() {
+    if (endpoint == null) {
+      throw new IllegalStateException("Endpoint is empty. Forget to configure \"loadbalance\" in consumer handler chain?");
```
Comment (WillemJiang) [inline core/src/main/java/io/servicecomb/core/Invocation.java]: Can we change the message to "Endpoint is empty. Please  configure \"loadbalance\" in the consumer handler chain.

Comment (eric-lee-ltk) [reply core/src/main/java/io/servicecomb/core/Invocation.java]: I guess this exception should be detected at startup time and should not be thrown at run time. Validate the `microservice.yaml` file before the application starts.

Comment (wujimin) [reply core/src/main/java/io/servicecomb/core/Invocation.java]: we don't know which handler is loadBalance
because cutomer can dev their own loadBalance

Comment (liubao68) [reply core/src/main/java/io/servicecomb/core/Invocation.java]: Yes, we don't know at which handler will a developer to set the address, and they can not use 'loadbalance' and define a address resolver themself. So this message is a best guess for the developer.

**Pull request merged**